Misc/NEWS.d/next/Core and Builtins/2017-09-26-13-03-16.bpo-31588.wT9Iy7.rst

```diff
@@ -0,0 +1,2 @@
+Raise a `TypeError` instead of `SystemError` in case class creation failed due
```

Comment:
I can't reproduce the reported `SystemError` myself - I get an unhelpfully cryptic error message, but it's a `TypeError` instance.

Reply:
I got the same message you got when I compiled in release mode. this is because checks that are in debug mode only, as I mentioned in https://bugs.python.org/issue31588#msg303122.

So without the patch we have a `TypeError` in release mode and a `SystemError` in debug mode.
I would change the phrasing of the NEWS.d item accordingly.